cumulus/parachains/integration-tests/emulated/common/src/lib.rs

```diff
@@ -49,6 +51,25 @@ pub const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
 
 type AccountPublic = <MultiSignature as Verify>::Signer;
 
+// This asset is added to AH as Asset and reserved transfer between Parachain and AH
+pub const RESERVABLE_ASSET_ID: u32 = 1;
+// This asset is added to AH as ForeignAsset and teleported between Penpal and AH
+pub const TELEPORTABLE_ASSET_ID: u32 = 2;
+
+pub const PENPAL_ID: u32 = 2000;
```

Comment:
we have also: https://github.com/paritytech/polkadot-sdk/pull/3543/files#diff-528b20b59bd1a15fbb063551a37e733ffa9b1e61c2d6ac30039e8d15271f5122R29-R30
```
pub const PARA_ID_A: u32 = 2000;
pub const PARA_ID_B: u32 = 2001;
```

so shouldn't be `PenpalTeleportableAssetLocation` and `PenpalSiblingSovereigAccount` specific for A and/or B?
E.g.:
`PenpalATeleportableAssetLocation`
`PenpalBTeleportableAssetLocation`